### ISSUE TITLE
Cactus pangenome workflow generates indexes not compatible with vg call

### ISSUE THREAD

**Issue by RenzoTale88** [ComparativeGenomicsToolkit/cactus#669]
Hello,
I've generate a graph genome using the cactus pangenome workflow for five insect assemblies, with small genome size (~230Mb).
The workflow completes successfully, and generates the indexes for giraffe (`--giraffe` flag active).
However, when I tried to run the `vg giraffe > vg pack > vg call` workflow, everything worked with the exception of the call function (see [here](https://github.com/vgteam/vg/issues/3551)).
After a bit of digging, I've found that the XG index did not contain any path. As explained [here](https://github.com/vgteam/vg/issues/3369), it looks like generating the XG using the initial GBWT/GG graphs generates and index which is not compatible with downstream analyses.
To avoid the issue, it is possible to run the `vg autoindex` function on the gfa generated by the pangenome workflow as follow:
```
vg autoindex -t8 -p GRAPHs/GIRAFFE -w giraffe --gfa GRAPH/final/5asm.gfa -R XG
```
This generates an XG which is fine to use for the `vg call` workflow.
Hope it can be of help.
Best regards
Andrea

**Comment by glennhickey** [comment]
Yeah, there are some fiddly naming conventions needed to get the paths right in the xg.  This is explained in more detail in the revised version of the pangenome README.  You can take a look in [this branch](https://github.com/ComparativeGenomicsToolkit/cactus/blob/mg-base/doc/pangenome.md#evolver-primates-preprocessing-and-naming) for now -- it will be in the next release (hopefully soon).  Note the commands in that document are not 100% compatible with the current release, but the bit on naming is appropriate.  

The important thing is that genome names follow a `SAMPLE.HAPLOTYPE` naming convention except the reference which is just `SAMPLE`.  For diploid samples the haplotype should be `1` or `2` and for haploid samples it should be `0`. 

If you've already created your alignment, you can rename the hal with `halRenameGenomes` or use the `--rename` option in `cactus-graphmap-join` to do it on the fly just before making the giraffe indexes.  

Sorry for the confusion -- hope this helps!
